test/tools/run.py

```diff
@@ -99,7 +99,7 @@ def create_device(config_file, org_id, project_id, device_name, userdata):
         f"Creating device {cfg['device']['name']} with config {pretty}")
 
     welder = Welder(token, cfg)
-    ip = welder.create_all()
+    ip = welder.create_all()[0]
```

Comment:
Can the call to `welder.create_all()` fail? Although this question has no bearing on the change.

Reply:
yes it can, but it has no impact on what that value is used for, as i have neglected to handle any errors properly and they just panic 😜